tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -9,235 +9,878 @@
+                    condition_function=validators.matches(1),
+                    result_field="CITIZENSHIP_STATUS",
+                    result_function=validators.oneOf((1, 2)),
+                ),
                 validators.if_then_validator(
-                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
-                        result_field='COOPERATION_CHILD_SUPPORT', result_function=validators.oneOf((1, 2, 9)),
-                    ),
+                    condition_field="FAMILY_AFFILIATION",
+                    condition_function=validators.isInLimits(1, 3),
+                    result_field="COOPERATION_CHILD_SUPPORT",
+                    result_function=validators.oneOf((1, 2, 9)),
+                ),
                 validators.validate__FAM_AFF__HOH__Fed_Time(),
                 validators.if_then_validator(
-                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
-                        result_field='EMPLOYMENT_STATUS', result_function=validators.isInLimits(1, 3),
-                    ),
+                    condition_field="FAMILY_AFFILIATION",
+                    condition_function=validators.isInLimits(1, 3),
+                    result_field="EMPLOYMENT_STATUS",
+                    result_function=validators.isInLimits(1, 3),
+                ),
                 validators.if_then_validator(
-                        condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
-                        result_field='WORK_ELIGIBLE_INDICATOR', result_function=validators.or_validators(
-                            validators.isInStringRange(1, 9),
-                            validators.oneOf(('11', '12'))
-                        ),
+                    condition_field="FAMILY_AFFILIATION",
+                    condition_function=validators.oneOf((1, 2)),
+                    result_field="WORK_ELIGIBLE_INDICATOR",
+                    result_function=validators.or_validators(
+                        validators.isInStringRange(1, 9), validators.oneOf(("11", "12"))
                     ),
+                ),
                 validators.if_then_validator(
-                        condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
-                        result_field='WORK_PART_STATUS', result_function=validators.oneOf(['01', '02', '05', '07', '09',
-                                                                                           '15', '17', '18', '19', '99']
-                                                                                          ),
+                    condition_field="FAMILY_AFFILIATION",
+                    condition_function=validators.oneOf((1, 2)),
+                    result_field="WORK_PART_STATUS",
+                    result_function=validators.oneOf(
+                        ["01", "02", "05", "07", "09", "15", "17", "18", "19", "99"]
```

Comment:
While I do like condensing this line, all these formatting changes drive up the chances of mistakes uncaught in review.